modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -80,10 +89,19 @@ type APIApplicationUIData = Pick<
 >;
 
 interface APIEndpointItem extends BaseItem {
-	name: string;
+	httpMethod: APIListType;
```

Comment:
Carlos suggested to rename 'HTTPMethod' to  'Action' since it is important to keep the parameter httpMethod as it is to match how the endpoint is defined on the backend;